### PR TITLE
fix: #666

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Fixed tickets    | #666 
| License          | MIT

### Description

[Description of the bug or feature]

--

### CODE REVIEW

File: src/renderer/components/titleBar.vue [hunk 1]
```diff
@@ -25,12 +25,14 @@
         class="frameless-titlebar-menu title-no-drag"
         @click.stop="handleMenuClick"
       >&#9776;</div>
-      <div
-        v-if="wordCount"
-        class="word-count"
-        :class="[{ 'title-no-drag': platform !== 'darwin' }]"
-        @click.stop="handleWordClick"
-      >{{ `${HASH[show]} ${wordCount[show]}` }}</div>
+      <el-tooltip class="item" effect="dark" :content="`${wordCount[show]} ${HASH[show].full + (wordCount[show] > 1 ? 's' : '')}`" :open-delay="1000" placement="bottom-end">
```
Comment (fxha) [inline src/renderer/components/titleBar.vue]: A line break would be highly appropriate here :smile:

Comment (fxha) [inline src/renderer/components/titleBar.vue]: I think we should decrease the open delay a bit.

Comment (Jocs) [reply src/renderer/components/titleBar.vue]: fixed

Comment (Jocs) [reply src/renderer/components/titleBar.vue]: yeh, I set it to 500 ms.

**Pull request merged**